package.json

```diff
@@ -59,20 +58,23 @@
     "@babel/preset-env": "7.25.3",
     "@babel/preset-react": "7.24.7",
     "@babel/preset-typescript": "7.24.7",
+    "@biomejs/biome": "1.9.2",
+    "@canonical/biome-config": "0.1.1-experimental.0",
```

Comment:
By adding Biome, you should be able to replace Eslint & prettier, but I don't see that happening in this PR, and biome isn't being called.

If this PR is scoped to just using the tsconfig, I'd suggest dropping the Biome dependencies. 

It's also worth nothing that Biome doesn't currently support Sass (and I see some of that in this project), so Biome couldn't fully replace Eslint & Prettier for this project.